src/platform/ESP32/nimble/BLEManagerImpl.cpp

```diff
@@ -56,1096 +56,1098 @@ using namespace ::chip;
 using namespace ::chip::Ble;
 
 namespace chip {
-namespace DeviceLayer {
```

Comment:
Why is the formatting of this file changed? As a result, so many changes are seen. Ideally, this file should have a single change, right?

Reply:
fixed